docs/configuration/alerting_rules.md

```diff
@@ -11,31 +11,36 @@ to an external service. Whenever the alert expression results in one or more
 vector elements at a given point in time, the alert counts as active for these
 elements' label sets.
 
-Alerting rules are configured in Prometheus in the same way as [recording
-rules](recording_rules.md).
-
 ### Defining alerting rules
 
-Alerting rules are defined in the following syntax:
-
-    ALERT <alert name>
-      IF <expression>
-      [ FOR <duration> ]
-      [ LABELS <label set> ]
-      [ ANNOTATIONS <label set> ]
-
-The alert name must be a valid metric name.
+Alerting rules are configured in Prometheus in the same way as [recording
+rules](recording_rules.md).
 
-The optional `FOR` clause causes Prometheus to wait for a certain duration
+A example rules file with an alert would be:
+
+```yaml
+groups:
+- name: example
+  rules:
+  - alert: HighErrorRate
+    expr: job:request_latency_seconds:mean5m{job="myjob"} > 0.5
+    for: 10m
+    labels:
+      severity: page
+    annotations:
+      summary: High request latency
+```
+
+The optional `for` clause causes Prometheus to wait for a certain duration
```

Comment:
This is a super long sentence with several concepts in it - aware it was here before - but perhaps broken up or punctuated?

Reply:
I don't see a way to break this up.

Reply:
Okay - I probably wasn't clear. The `for` clause is a flapping check right? It is a way to ensure the alert actually fires for a real error? That's not clear to me from that sentence. Perhaps...

```
The optional `for` clause causes Prometheus to wait for a specified duration between first encountering a new expression output vector element and counting an alert as firing for this element. Here Prometheus will wait 10 minutes after the first element... etc etc...
```

Or apologies if I have misread what `for` is ... for. :)

Reply:
That's not really any shorter.

The current description isn't 100% correct, but I this PR is more about getting us basically in shape rather than fixing all the definitions.